### DISCUSSION TITLE
AH01067: Failed to read FastCGI header | kimai kimai 1.30.1

### DISCUSSION THREAD

**Discussion by GaneshKandu** [kimai/kimai#3719] [On-Premise: Installation & Updates] [answered]
### Describe the issue

Hi,

I'm getting ``503 Service Unavailable error`` after install of Kimai 1.30.1 on ``en/dashboard`` page.
all other urls are working, except ``en/dashboard``

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

1.30.1

### Which PHP version are you using?

7.4

### How do you run Kimai?

Self-Hosted

### Steps to reproduce the behavior

Followed this steps [https://www.kimai.org/documentation/installation.html](https://www.kimai.org/documentation/installation.html) till ``Create your first user``

1. Go to login page and login
2. Click on ``Dashboard``
3. See error

## Kimai log
No Error in Kimai log

## Apache log 

```
[Fri Dec 30 03:10:44.358836 2022] [proxy_fcgi:error] [pid 2755686] [client xxx.xxx.xxx.xxx:32094] AH01067: Failed to read FastCGI header, referer: http://mydomain.com/en/team/timesheet/
[Fri Dec 30 03:10:44.358880 2022] [proxy_fcgi:error] [pid 2755686] (104)Connection reset by peer: [client xxx.xxx.xxx.xxx:32094] AH01075: Error dispatching request to : , referer: http://mydomain.com/en/team/timesheet/
[Fri Dec 30 04:00:14.306856 2022] [proxy_fcgi:error] [pid 2755682] [client xxx.xxx.xxx.xxx:15163] AH01067: Failed to read FastCGI header, referer: http://mydomain.com/en/login
[Fri Dec 30 04:00:14.307725 2022] [proxy_fcgi:error] [pid 2755682] (104)Connection reset by peer: [client xxx.xxx.xxx.xxx:15163] AH01075: Error dispatching request to : , referer: http://mydomain.com/en/login
[Fri Dec 30 04:00:16.688147 2022] [proxy_fcgi:error] [pid 2755683] [client xxx.xxx.xxx.xxx:27529] AH01067: Failed to read FastCGI header, referer: http://mydomain.com/en/login
[Fri Dec 30 04:00:16.688178 2022] [proxy_fcgi:error] [pid 2755683] (104)Connection reset by peer: [client xxx.xxx.xxx.xxx:27529] AH01075: Error dispatching request to : , referer: http://mydomain.com/en/login
[Fri Dec 30 04:00:18.851835 2022] [proxy_fcgi:error] [pid 2755773] [client xxx.xxx.xxx.xxx:32862] AH01067: Failed to read FastCGI header, referer: http://mydomain.com/en/login
[Fri Dec 30 04:00:18.851877 2022] [proxy_fcgi:error] [pid 2755773] (104)Connection reset by peer: [client xxx.xxx.xxx.xxx:32862] AH01075: Error dispatching request to : , referer: http://mydomain.com/en/login
[Fri Dec 30 04:00:20.923439 2022] [proxy_fcgi:error] [pid 2755774] [client xxx.xxx.xxx.xxx:29209] AH01067: Failed to read FastCGI header, referer: http://mydomain.com/en/login
[Fri Dec 30 04:00:20.923472 2022] [proxy_fcgi:error] [pid 2755774] (104)Connection reset by peer: [client xxx.xxx.xxx.xxx:29209] AH01075: Error dispatching request to : , referer: http://mydomain.com/en/login
[Fri Dec 30 04:00:23.113634 2022] [proxy_fcgi:error] [pid 2755689] [client xxx.xxx.xxx.xxx:15663] AH01067: Failed to read FastCGI header, referer: http://mydomain.com/en/login
[Fri Dec 30 04:00:23.113672 2022] [proxy_fcgi:error] [pid 2755689] (104)Connection reset by peer: [client xxx.xxx.xxx.xxx:15663] AH01075: Error dispatching request to : , referer: http://mydomain.com/en/login
```

### Screenshots

![image](https://user-images.githubusercontent.com/12833130/210054535-3025b74d-4316-4c40-b163-5e52b1db3652.png)

### Additional context

Device: Linux 3.10.0-962.3.2.lve1.5.26.9.el7.x86_64
Browser Firefox 108.0.1

I'm getting this error on CPanel installed Server only ( Reproduced on multiple CPanel Installed Servers )
On other panels its working.

This error occures when in production mode ``APP_ENV=prod``
But working when in dev mode ``APP_ENV=dev``

**Comment by kevinpapst** [comment, marked as answer]
I believe that you see this problem, but from your description I can only say that this is a CPanel problem.
I will convert this into a discussion, as it is definitely not a Kimai bug.

I'd love to help, but without any indication what this might be causing, I can't do anything.

Does that happen when using PHP 8 or 8.1 or 8.2 as well?

**Reply by GaneshKandu** [reply]
> Does that happen when using PHP 8 or 8.1 or 8.2 as well?

Its working on PHP 8 and 8.1

**Reply by kevinpapst** [reply]
One year ago, end of support for 7.x was announced: https://www.kimai.org/blog/2021/sunsetting-php-7/

PHP 7.4 is official EOL, see https://www.php.net/supported-versions.php

A problem that happens only on CPanel and only on a EOL PHP Version... what exactly do you expect me to do here?

**Reply by GaneshKandu** [reply]
> what exactly do you expect me to do here?

You should remove PHP 7.4 from minimum required PHP version
https://github.com/kimai/kimai#requirements

**Reply by kevinpapst** [reply]
Aha, thanks! 
That will be removed with the new major version, which is coming very soon. See #2902 

AFAIK 7.4 working totally fine and even 7.3 is working, I just cannot support these EOL versions.
